mongo/options/createcollectionoptions.go

```diff
@@ -28,18 +30,35 @@ func (d *DefaultIndexOptions) SetStorageEngine(storageEngine interface{}) *Defau
 
 // TimeSeriesOptions specifies options on a time-series collection.
 type TimeSeriesOptions struct {
-	// Name of the top-level field to be used for time. Inserted documents must have this field,
+	// TimeField is the top-level field to be used for time. Inserted documents must have this field,
 	// and the field must be of the BSON UTC datetime type (0x9).
 	TimeField string
 
-	// Optional name of the top-level field describing the series. This field is used to group
+	// MetaField is the name of the top-level field describing the series. This field is used to group
```

Comment:
Updated the comments to start with the name of the field. This is a common practice in Go libraries, e.g

- https://pkg.go.dev/net/http#Client
- https://pkg.go.dev/google.golang.org/grpc#ClientConnInterface

This is also a common practice in the Go Driver.